docusaurus/docs/React/release-guides/new-reactions.mdx

```diff
@@ -0,0 +1,135 @@
+# Introducing new reactions (with breaking changes)
+
+As a "move-away-from-EmojiMart" initiative we decided to start with reaction components (selector and list) and revamp the DX to make customization of these components much easier and more intuitive without having to rebuild default selector and list components from scratch.
```

Comment:
I think, it would help those not well versed in the topic to provide a link to doc, that explains, how reactions work in the SDK.